Gcp-ace-section4-complete-guide.html

```
@@ -1178,15 +1230,40 @@ <h1 class="hero-title">
 <span class="comment"># AWS: 特定の IAM ロールのみ許可</span>
 --attribute-condition="attribute.aws_role == 'arn:aws:sts::ACCOUNT:assumed-role/ROLE'"</pre>
 
+    <div class="sub-title">AWS との設定例</div>
+    <div class="callout callout-blue"><span class="callout-icon">☁️</span><div>AWS EC2 や Lambda 等から SA キーなしで GCP リソースにアクセスする場合は AWS Provider を使用します。AWS IAM ロールの ARN を attribute-condition で限定することが必須です。</div></div>
+    <pre style="position:relative;"><button class="copy-btn" onclick="copyCode(this)">Copy</button><span class="comment"># AWS 用 Workload Identity Pool Provider を作成</span>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)